src/purchases.ts

```diff
@@ -922,7 +922,7 @@ export default class Purchases {
    * @returns {Promise<void>} The promise will be rejected if there's an error setting the proxy url.
```

Comment:
Maybe I'm misunderstanding, but is this statement true? Is the promise being rejected in case of an error?

Reply:
Well, this should normally never fail so yeah, we should update these docs 👍 Good thinking!